cruise/CruiseInformation.csv

```diff
@@ -1,158 +1,161 @@
+EK20180824,R/V Elakha,2018-08-24T15:45:00,2018-08-24T19:38:00,Endurance Glider deployment 319-00007
+ML20180907,Miss Linda,2018-09-07T13:29:00,2018-09-07T17:24:00,Endurance Glider Deployment 382-00004
+SR1813,R/V Sally Ride,2018-09-17T11:30:00,2018-10-02T14:20:00,Endurance 10 and cspps and gliders
+EK20181017,R/V Elakha,2018-10-17T15:15:00,2018-10-17T21:20:00,Endurance Glider operations
+AR-31,R/V Neil Armstrong,2018-10-20T18:00:00,2018-11-09T16:30:00,Pioneer 11
+EK20181022,R/V Elakha,2018-10-22T11:49:00,2018-10-22T15:00:00,Endurance Glider recovery
+SU20181101,R/V Pacific Surveyor,2018-11-01T15:10:00,2018-11-01T18:22:00,Endurance Glider Deployment
+MR20181116,F/V Miss Raven,2018-11-16T16:14:00,2018-11-16T18:54:00,Endurance Glider Recovery
+DY096,RRS Discovery,2018-11-28T09:00:00,2018-12-14T11:10:00,Southern Ocean 5 (CUSTARD)
+TN20181130,F/V Tenacious II,2018-11-30T08:00:00,2018-11-30T23:59:00,Pioneer Glider Deploy - 339 deploy 8 by Eric Hesse #TO BE DELETED
+TS20181130,F/V Tenacious II,2018-11-30T08:00:00,2018-11-30T23:59:00,Pioneer Glider Deploy - 339 deploy 8 by Eric Hesse
+ML20181203,Miss Linda,2018-12-03T16:15:00,2018-12-03T18:42:00,Endurance Glider Recovery
+RC20181203,R/V Rachel Carson,2018-12-03T18:00:00,2018-12-08T21:00:00,Endurance Profiler Recovery and Glider Deployment
+TN20181209,F/V Tenacious II,2018-12-09T05:00:00,2018-12-09T23:59:00,Glider 339 deploy 9 by Eric Hesse #TO BE DELETED
+TS20181209,F/V Tenacious II,2018-12-09T05:00:00,2018-12-09T23:59:00,Glider 339 deploy 9 by Eric Hesse
+OC1901A,R/V Oceanus,2019-01-11T00:00:00,2019-01-15T00:00:00,Opportunity glider recovery by Oceanus martech
+SU20190125,R/V Pacific Surveyor,2019-01-25T16:30:00,2019-01-26T04:26:00,Endurance Glider Deployments
+WJ20190203,M/V Warren Jr.,2019-02-03T09:00:00,2019-02-04T21:00:00,"Pioneer 11a, OSSM D9 buoy recovery, GL388 D7 recovery"
+TS20190205,F/V Tenacious II,2019-02-05T00:30:00,2019-02-06T01:00:00,"Pioneer 11b, glider deployments"
+TY-1902,CCGS Tully,2019-02-05T12:00:00,2019-02-23T12:00:00,Papa Glider gp_276 recovery on the Line P 2019-002 cruise
+WJ20190228,M/V Warren Jr.,2019-02-28T17:00:00,2019-03-02T13:00:00,"Pioneer 11c, AUV deployments, glider cp_339 recovery"
+EK20190403,R/V Elakha,2019-04-03T15:00:00,2019-04-03T22:00:00,ce01issp_00012 and ce02shsp_00012 deployments
+AR-34,R/V Neil Armstrong,2019-04-04T15:10:00,2019-04-24T11:36:00,Pioneer 12
+ML20190417,R/V Miss Linda,2019-04-07T15:07:00,2019-04-17T22:00:00,Endurance glider ce_327 recovery
+SKQ201910S,R/V Sikuliaq,2019-04-19T20:00:00,2019-04-28T15:46:47,Endurance 11A
+OC1905C,R/V Oceanus,2019-05-03T20:00:00,2019-05-08T00:00:00,Endurance 11B
+BK20190504,M/V Blackhawk,2019-05-04T09:00:00,2019-05-04T20:00:00,Pioneer cp_340 recovery
+TS20190516,F/V Tenacious II,2019-05-16T12:00:00,2019-05-17T12:00:00,Pioneer 12a glider deployment
+SKQ20190514,R/V Sikuliaq,2019-05-14T00:00:00,2019-05-27T00:00:00,Papa Glider gp_469 recovery on the San Diego - Seward transit
```

Comment:
This cruise number does not look correct.  Based on the Sikuliaq's cruise id scheme and the current cruise (SKQ201914S), I think it should be SKQ201913T.  If no one knows, maybe Dee can help to confirm.

Reply:
@snwhite I confirmed that SKQ201913T is the correct cruise ID. Good catch!